package.json

```diff
@@ -33,7 +33,9 @@
   },
   "scripts": {
     "build": "rollup -c",
-    "build:test": "REACT_APP_TEST=true yarn build",
+    "build:test": "yarn build --vegaEnv=test",
```

Comment:
Either use an arg like `--vega-env` or perhaps something like `--config` (if that collides with rollup perhaps `--wallet-config`)